webrender/src/texture_cache.rs

```diff
@@ -594,6 +600,42 @@ impl TextureCache {
         }
     }
 
+    pub fn insert_image_border_updating_operation(src: &[u8],
+                                                  alloc_x: u32,
+                                                  alloc_y: u32,
+                                                  alloc_width: u32,
```

Comment:
let's use rectangles (`DeviceUintRect`) here for both the allocation and request regions